js/vendor/owncloud/share.js

```diff
@@ -145,7 +145,7 @@
 			var dropDownEl;
 			var self = this;
 			var html = '<div id="dropdown" class="drop shareDropDown" data-item-type="' + itemType +
-				'" data-item-source="' + path + '">';
+				'" data-item-source="' + path + '" data-id="' + data[0].file_source + '">';
```

Comment:
What happens when you open an album which has never been shared? I think you won't be able to send an email.